cmd/main.go

```diff
@@ -93,6 +93,24 @@ func init() {
 	}
 }
 
+func getTerminalDimensions() (int, int, error) {
+	tty, err := os.Open("/dev/tty")
+	isTTY := true
+	if err != nil {
+		tty = os.Stdin
+		isTTY = false
+	}
+	if !isTTY {
+		return -1, -1, errors.New("not a tty device")
+	}
+	ws, err := unix.IoctlGetWinsize(int(tty.Fd()), unix.TIOCGWINSZ)
```

Comment:
you need a platform-independent function here @vadmeste such `x/term.GetSize(tty.Fd())`

Reply:
> you need a platform-independent function here @vadmeste such x/term.GetSize(tty.Fd())

yes, thanks for the suggestion